receiver/hostmetricsreceiver/internal/scraper/loadscraper/load_scraper_windows.go

```diff
@@ -140,7 +140,7 @@ func stopSampling(_ context.Context) error {
 		// no load scraper is running nothing to do
 		return nil
 	}
-	
+
```

Comment:
this change seems unrelated

Reply:
I did not chang it manually, seems changed by `make genotelcontribcol  `

Reply:
Do I need  revert this line ?

Reply:
No, I think this is not a big deal